### PR TITLE
More info in text only email

### PR DESCRIPTION
**Pull request by jSylvestre**
For email sent to person who has expiring items.

### CODE REVIEW

File: Keas.Core/Services/IEmailService.cs [hunk 1]
```diff
@@ -298,6 +299,53 @@ public async Task SendExpiringMessage(int personId, ExpiringItemsEmailModel mode
...
+            var message = new StringBuilder();
+            var count = expiringItems.KeySerials.Count() + expiringItems.Equipment.Count() + expiringItems.Workstations.Count() + expiringItems.AccessAssignments.Count();
+            var helplink = "https://peaks.ucdavis.edu/" + expiringItems.Person.Team.Slug +"/Help";
+
+            var plural = "item";
+            if (count > 1)
+            {
+                plural = "items";
+            }
+            message.AppendLine("It looks like you are viewing the ugly text version of this email, if you view it in html it will look a lot nicer.");
```
Comment (srkirkland) [inline Keas.Core/Services/IEmailService.cs]: nope not the hill i'm going to die on

Comment (jSylvestre) [reply Keas.Core/Services/IEmailService.cs]: image

**Comment by jSylvestre** [comment]
* [ ] Manually deploy email job when this is merged

**Comment by jSylvestre** [comment]
`--_----PDHJFU2/jsOAe07VvaMb9Q===_C2/FC-27452-03A9A9C5
Content-Transfer-Encoding: 7bit
Content-Type: text/plain; charset="UTF-8"

Jason Sylvestre, you have 2 expiring items in team notify test in PEAKS.
Thanks for PEAKing at this email. It sPEAKS well of you.

Notification Details
Type, Item, Date expiring
Access, test 1, 03/26/2019
Equipment, test 2, 03/26/2019

Please contact your team's assigned individuals to either update the expiration date or return/check-in the above items.

Not sure who that is? Click Help. To see your team's Admins
Help link for this team: http://goto.ucdavis.edu/f/a/nkPixQoR-Z5Hxe-wY8PXCw~~/AAB5cQA~/RgRefR8wP0QqaHR0cHM6Ly9wZWFrcy51Y2RhdmlzLmVkdS9ub3RpZnktdGVzdC9IZWxwVwNzcGNCCgAlMJqaXDxrPc9SGWphc29uY3N5bHZlc3RyZUBnbWFpbC5jb21YBAAAAlM~

This email was automatically generated please do not reply to it as the mailbox is not monitored.`

**Comment by jSylvestre** [comment]
Content-Transfer-Encoding: 7bit
Content-Type: text/plain; charset="UTF-8"

Jason Sylvestre, you have 8 new notifications from PEAKS.
Thanks for PEAKing at this email. It sPEAKS well of you.

Notification Details
Team: Test bad user
Details, Date Created
KeySerial(Cebra Lab: 385 through 397A) Unassigned to Samuel Heppelmann (sheppel)  by Jason Sylvestre (jsylvest), 03/18/2019 10:58 AM
KeySerial(Cebra Lab: 385 through 397A) Unassigned to Samuel Heppelmann (sheppel)  by Jason Sylvestre (jsylvest), 03/18/2019 10:58 AM
Team: notify test
One or more items in this team require you to accept them. Please click on the link below to confirm receiving these items.
Link to confirm: http://goto.ucdavis.edu/f/a/nqgJla2S3PY3cMYnBl2XSg~~/AAB5cQA~/RgRefR8vP0Q1aHR0cHM6Ly9wZWFrcy51Y2RhdmlzLmVkdS9ub3RpZnktdGVzdC9Db25maXJtL0NvbmZpcm1XA3NwY0IKAB4vmppciUKGHFIZamFzb25jc3lsdmVzdHJlQGdtYWlsLmNvbVgEAAACUw~~
Details, Date Created
Access(test 1) Created by Jason Sylvestre (jsylvest), 03/26/2019 2:25 PM
Access(test 1) Assigned to Jason Sylvestre (jsylvest)  by Jason Sylvestre (jsylvest), 03/26/2019 2:25 PM
Access(test 1) Assigned to Jason Sylvestre (jsylvest)  by Jason Sylvestre (jsylvest), 03/26/2019 2:25 PM
Equipment(test 2) Created by Jason Sylvestre (jsylvest), 03/26/2019 2:26 PM
Equipment(test 2) Assigned to Jason Sylvestre (jsylvest)  by Jason Sylvestre (jsylvest), 03/26/2019 2:26 PM
Equipment(test 2) Assigned to Jason Sylvestre (jsylvest)  by Jason Sylvestre (jsylvest), 03/26/2019 2:26 PM

This email was automatically generated please do not reply to it as the mailbox is not monitored.

**Pull request merged**